orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/WaitStage.java

```diff
@@ -33,44 +33,33 @@
   public static String STAGE_TYPE = "wait";
 
   @Override
-  public void taskGraph(Stage stage, TaskNode.Builder builder) {
+  public void taskGraph(@NotNull Stage stage, TaskNode.Builder builder) {
```

Comment:
The super method uses `@Nonnull` from `javax.annotation`, which is in general the non-null annotation we use more commonly (including elsewhere in this class) so I'd suggest using that here.

Reply:
silly me for blindly accepting intelliJ offer.. Thanks for catching it!